src/quacc/recipes/espresso/dos.py

```diff
@@ -165,6 +165,7 @@ def dos_flow(
         Dictionary of results from [quacc.schemas.ase.summarize_run][].
         See the type-hint for the data structure.
     """
+    job_params = job_params or {}
```

Comment:
Strictly speaking, these changes are likely not needed because `recursive_dict_merge` called [here](https://github.com/AitElBadaoui/quacc/blob/5bf34c17dc1365ddceb4690f47f00e3e961ff7ac/src/quacc/wflow_tools/customizers.py#L178) will treat `None` as `{}`. I think?

Reply:
I added this line because the `job_params` is used in [(line 175)](https://github.com/AitElBadaoui/quacc/blob/5bf34c17dc1365ddceb4690f47f00e3e961ff7ac/src/quacc/recipes/espresso/dos.py#L175) and since the function signature allows it to be None, we need to initialize the variable if it's None.

Reply:
Excellent find!